### DISCUSSION TITLE
Feature Request: Integration with Apollo Router

### DISCUSSION THREAD

**Discussion by oo00spy00oo** [graphql-hive/console#159] [Feature request]
The next version of Apollo Gateway is Apollo Router with description here: https://www.apollographql.com/docs/router

Need a plugin for integrate between Hive and Apollo Router.

Regards.

**Comment by kamilkisiela** [comment]
Yes! That's definitely something we want to implement

**Comment by kamilkisiela** [comment]
I added it to our roadmap https://github.com/kamilkisiela/graphql-hive/issues/255

**Comment by kamilkisiela** [comment]
https://docs.graphql-hive.com/features/registry-usage#apollo-router
and
https://docs.graphql-hive.com/features/monitoring#apollo-router
